README.md

```diff
@@ -144,9 +144,11 @@ dependencies:
       revision: master
   systems:  # An optional list of repos that contain an opencontrol.yaml stored remotely
     - url: https://github.com/18F/cg-compliance
+      contextdir: 18F     # An optional ket to specify a subdirectory in GIT
       revision: master
   standards:   # An optional list of remote repos containing standards info that contain an opencontrol.yaml
     - url: https://github.com/opencontrol/NIST-800-53-Standards
+      contextdir: nist    # An optional ket to specify a subdirectory in GIT
```

Comment:
Yes - realize this expanding scope beyond your original PR - but could you update that URL to https://github.com/opencontrol/standards ?

Reply:
That project does not next it's content into a subdirectory. Is ``contextdir`` needed?